terraform/deployments/ga4-analytics/service_account.tf

```diff
@@ -0,0 +1,33 @@
+resource "google_service_account" "sa--ga4-analytics-352613" {
+  account_id                   = "ga4-analytics-352613"
+  disabled                     = "false"
+  display_name                 = "App Engine default service account"
+  project                      = google_project.project.project_id
```

Comment:
Can remove project as defaults to the one used by the provider.

Reply:
thanks, done